dvzo/settings.py

```diff
@@ -23,18 +23,19 @@
 ]
 
 INSTALLED_APPS = [
-    'whitenoise.runserver_nostatic',
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    'django_tex',
-    'phonenumber_field',
-    'tapeforms',
-    'compressor',
-] + PROJECT_APPS
+                     'whitenoise.runserver_nostatic',
```

Comment:
Warum hast du die indentation geändert? Das macht es viel schwieriger ein Review zu machen und die git history für die anderen Zeilen wird unübersichtlicher.

Reply:
hab die einrückung in einem anderen teil des files angepasst und da hat es offensichtlich alles angepasst. kann ich gerne rückgängig machen

Reply:
Das kann man ausschalten. Und falls du das nicht willst würde ich vorschlagen black oder so zu verwenden für die Formatierung.
Ich würde es nicht zu ändern versuchen, ausser du willst dich in git rebase üben. Aber es befindet sich im ersten commit...